.gitignore

```diff
@@ -7,6 +7,3 @@ __pycache__
 # .tfstate files
 *.tfstate
 *.tfstate.*
-
-# Compiled JSonnet templates
-configs/jsonnet/*
```

Comment:
We should ignore these so we don't pollute the diffs. Most of the tests generated from the other repository are not used here.

Reply:
Oh, I see what's wrong. The auto-deployment pipeline doesn't build the tests. Let me see if there's a snippet you can copy/paste to build the JSonnet templates as part of the deployment pipeline. We build the templates [dynamically for GHA](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/bdb22c31138ff1f81506ea3d123c13864c4a8a83/.github/workflows/dag-check.yml#L18-L35), but the environment is a little different for GCB. I can take a look tomorrow if there's no easy fix.

Reply:
Thanks for the fix! Removed Jsonnet configs in this PR.